scripts/helpers/db-test-helpers.ts

```diff
@@ -65,6 +65,7 @@ export async function recreateTestDbTemplate(
     dbConfig.dbLoginPassword,
     dbConfig.dbOwner,
     dbConfig.dbOwnerPassword,
+    true,
```

Comment:
to follow the approach with db:reset adjustments, this should also be a
parameter, so that e.g. `test:reset:dbs` would not create test owner users with
replication attribute

Reply:
The outbox/inbox PR will not use logical replication but the polling listener approach.